controllers/volumereplication_controller.go

```diff
@@ -84,7 +84,7 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			// Request object not found, could have been deleted after reconcile request.
 			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
 			// Return and don't requeue
-			r.Log.Info("no VolumeReplication resource found")
```

Comment:
(question) If you actually used the logger returned from line 77 it would print this information for all logs, without having to add this to each log line. Attempting to learn why that was not chosen.

Reply:
Yeah, Nice question. line 77 did not caught my attention. Maybe we should use that. @umangachapagain @sp98 Can we use that or there was any specific reason to not use it?

Reply:
We should use that.

Reply:
Done